### PR TITLE
Allow Symfony 6

### PR DESCRIPTION
**Pull request by simPod**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
         "ext-phar": "*",
         "composer-runtime-api": "^2.0.0",
         "nikic/php-parser": "^4.13.0",
-        "symfony/console": "^5.4.0",
+        "symfony/console": "^5.4.0 || ^6.0.0",
```
Comment (Ocramius) [inline composer.json]: IMO just drop 5.4.x here: this is an application, not a library.

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (simPod) [reply composer.json]: 👍🏾

**Comment by simPod** [comment]
Mkay, this requires more dep updates, see the diff

**Comment by simPod** [comment]
I have done standalone psalm here ~https://github.com/maglnet/ComposerRequireChecker/pull/338~ https://github.com/maglnet/ComposerRequireChecker/pull/339

What about php version? We can either have

```
php 7.4 | 8
symfony 5.4 | 6
```

or

```
php 8
symfony 6
```

**Comment by Ocramius** [comment]
@simPod you know me well: just kill PHP 7 :P

**Review by Ocramius — APPROVED** [review]
:ship: 

Thanks @simPod!

**Pull request merged**